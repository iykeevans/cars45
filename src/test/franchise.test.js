import { render } from "./test-utils";

import Franchise from "../pages/franchise";

describe("Franchise Page", () => {
  test("Franchise page matches snapshot", () => {
    const { asFragment } = render(<Franchise />);
    expect(asFragment()).toMatchSnapshot();
  });
});
