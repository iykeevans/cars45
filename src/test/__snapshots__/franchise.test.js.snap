// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Franchise Page Franchise page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="header-one"
  >
    <div
      class="container"
    >
      <div
        class="row justify-content-md-end"
      >
        <div
          class="col-10 col-lg-4 phone-box"
        >
          <div
            class="top-phone"
          >
            <img
              alt="phone"
              src="/assets/icons/phone.svg"
            />
            <p
              class="darkgrey-color"
            >
              +234 818 984 0160
            </p>
            <div
              class="divider"
            />
            <p
              class="darkgrey-color"
            >
              +234 818 984 0160
            </p>
          </div>
        </div>
        <div
          class="col-1 col-md-1 profile"
        >
          <div
            class="btn-group"
          >
            <button
              aria-expanded="false"
              aria-haspopup="true"
              class="btn btn-outline-secondary dropdown-toggle"
              data-toggle="dropdown"
              type="button"
            >
              <img
                alt="profile"
                src="https://storage.googleapis.com/cars45-web-bucket/profile-icon.svg"
              />
            </button>
            <div
              class="dropdown-menu"
            >
              <button
                class="dropdown-item btn btn-link darkgrey-color"
              >
                Sign In
              </button>
              <div
                class="dropdown-divider"
              />
              <button
                class="dropdown-item btn btn-link darkgrey-color"
              >
                Register
              </button>
            </div>
          </div>
        </div>
      </div>
      <nav
        class="navbar navbar-expand-lg navbar-light bg-light"
      >
        <a
          class="navbar-brand"
          href="/"
        >
          <img
            alt="logo"
            class="logo"
            src="https://storage.googleapis.com/cars45-web-bucket/logo.svg"
          />
        </a>
        <button
          aria-controls="navbarNavDropdown"
          aria-expanded="false"
          aria-label="Toggle navigation"
          class="navbar-toggler"
          data-target="#navbarNavDropdown"
          data-toggle="collapse"
          type="button"
        >
          <span
            class="navbar-toggler-icon"
          />
        </button>
        <div
          class="collapse navbar-collapse"
          id="navbarNavDropdown"
        >
          <ul
            class="navbar-nav"
          >
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/brandnew"
              >
                Brand New Cars 
              </a>
            </li>
            <li
              class="nav-item dropdown"
            >
              <button
                aria-expanded="false"
                aria-haspopup="true"
                class="nav-link dropdown-toggle btn btn-outline-secondary used"
                data-toggle="dropdown"
                id="navbarDropdownMenuLink"
                role="button"
              >
                Used Cars
              </button>
              <div
                aria-labelledby="navbarDropdownMenuLink"
                class="dropdown-menu used-menu"
              >
                <a
                  class="dropdown-item"
                  href="/buy"
                >
                  Buy
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/sell"
                >
                  Sell
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/swap"
                >
                  Swap
                </a>
              </div>
            </li>
            <li
              class="nav-item dropdown"
            >
              <button
                aria-expanded="false"
                aria-haspopup="true"
                class="nav-link dropdown-toggle btn btn-outline-secondary inspection"
                data-toggle="dropdown"
                id="navbarDropdownMenuLink"
                role="button"
              >
                Inspection Services
              </button>
              <div
                aria-labelledby="navbarDropdownMenuLink"
                class="dropdown-menu inspect"
              >
                <a
                  class="dropdown-item"
                  href="/ride-hailing"
                >
                  Ride Hailing 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/private-individual"
                >
                  Private/Individual 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/premium-inspection"
                >
                  Premium Inspection 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/valuation"
                >
                  Valuation 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/due-dilligence"
                >
                  Due Dilligence 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/all-in-one"
                >
                  All-in-one 
                </a>
              </div>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/loan"
              >
                Car Loan 
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/service"
              >
                Service & Repair 
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/franchise"
              >
                Become a Franchise 
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/corporate"
              >
                Corporate Service 
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/import"
              >
                Import Cars 
              </a>
            </li>
            <li
              class="nav-item"
            >
              <a
                class="nav-link"
                href="/referral"
              >
                Referral Program 
              </a>
            </li>
            <li
              class="nav-item dropdown"
            >
              <button
                aria-expanded="false"
                aria-haspopup="true"
                class="nav-link dropdown-toggle btn btn-outline-secondary"
                data-toggle="dropdown"
                id="navbarDropdownMenuLink"
                role="button"
              >
                Other Services
              </button>
              <div
                aria-labelledby="navbarDropdownMenuLink"
                class="dropdown-menu"
              >
                <a
                  class="dropdown-item"
                  href="/about"
                >
                  About Us 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/check"
                >
                  Background Check 
                </a>
                <div
                  class="dropdown-divider"
                />
                <a
                  class="dropdown-item"
                  href="/dealer"
                >
                  Become A Dealer 
                </a>
              </div>
            </li>
          </ul>
        </div>
      </nav>
    </div>
  </div>
  <div
    class="dealer-container"
  >
    <div
      class="dealer-banner"
    >
      <h1
        class="header"
      />
    </div>
    <div
      class="dealer-intro"
    >
      <p>
        Driven by a mission to build an ecosystem that enhances,enables and
  drives trade within Nigeria's automotive sector, Cars45 is Nigeria's
  largest digital automotive trading platform. We are a go-to platform
  providing people and businesses with a convenient and reliable way to
  buy, sell and swap used cars using modern innovative and
  technologically driven processes.
      </p>
    </div>
    <div
      class="dealer-commitment"
    >
      <h1>
        We Are Highly Committed
      </h1>
      <p>
        We have an unparalleled commitment to quality and our innovative, evolving product and service offerings attest to this.
        <br />
         With the support of Cars45 Management and Staff, we will keep you going for SUCCESS.
      </p>
      <h5
        class="motivation"
      >
        “ Motivation gets you started; Commitment keeps you going ”
      </h5>
      <p>
        We provide the safest marketplace for you. Access to our inventory of certified cars.
        <br />
         Verified Documents and paperwork. Simple Car Transfer and payment.
      </p>
      <div
        class="line"
      />
      <h1>
        The BIG Idea – Project F.A.S.T
      </h1>
      <div
        class="dealer-details"
      >
        <div
          class="dealer-goals dealer-orange"
        >
          <h6>
            A multi level Incentive based program as it aim to:
          </h6>
          <ul
            class="points"
          >
            <li>
              Grow your business
            </li>
            <li>
              Provide YOU with prime support
            </li>
            <li>
              Recognize and reward performance
            </li>
            <li>
              Promote healthy competition
            </li>
          </ul>
        </div>
        <div
          class="dealer-goals dealer-red"
        >
          <h6>
            An overall business model: highly profitable and efficient a it would ascertain:
          </h6>
          <ul
            class="points"
          >
            <li>
              Brand transformation
            </li>
            <li>
              Improved customer service
            </li>
            <li>
              Impactful advertising and digital marketing footprint
            </li>
            <li>
              Strong, experienced leadership
            </li>
          </ul>
        </div>
      </div>
      <div
        class="dealer-points"
      >
        <div
          class="dealer-point-meet"
        >
          <div
            class="dealer-point-container"
          >
            <div
              class="dealer-point-box"
            >
              <p>
                Build Your Brand
              </p>
            </div>
            <div
              class="dealer-point-line"
            />
          </div>
          <div
            class="dealer-point-container"
          >
            <div
              class="dealer-point-box"
            >
              <p>
                Improve Profitable Sales
              </p>
            </div>
            <div
              class="dealer-point-line"
            />
          </div>
          <div
            class="dealer-point-container"
          >
            <div
              class="dealer-point-box"
            >
              <p>
                Build a Sustainable Business
              </p>
            </div>
            <div
              class="dealer-point-line"
            />
          </div>
          <div
            class="dealer-point-container"
          >
            <div
              class="dealer-point-box"
            >
              <p>
                Discounted Retainership Agreement
              </p>
            </div>
            <div
              class="dealer-point-line"
            />
          </div>
        </div>
        <p
          class="dealer-point-statement"
        >
          WITH LOVE FROM US TO YOU
        </p>
      </div>
    </div>
    <div
      class="dealer-journey"
    >
      <h1>
        Start Your Journey
      </h1>
      <ul>
        <li>
          Have little or no Car Dealership experience and/or interested in
    owning a lucrative retail franchise
        </li>
        <li>
          Extensive Car dealership experience, with car lots in one or
    several locations and would like to join our network
        </li>
        <li>
          Commitment to operational excellence, customer service and a
    strong willingness to learn and practice Cars45 Business Model
        </li>
        <li>
          Demonstrate exceptional managerial and business acumen
        </li>
      </ul>
    </div>
    <div
      class="dealer-form-field"
    >
      <form
        class="dealer-form"
      >
        <a
          href="https://app.pipefy.com/public/form/yP4VaDaG"
        >
          Sign Up
        </a>
      </form>
      <div
        class="color-footer"
      />
    </div>
    <div
      class="deal-contact-details"
    >
      <div
        class="deal-contact-security"
      >
        <img
          alt="security"
          class="mr-2"
          src="https://storage.googleapis.com/cars45-web-bucket/security.svg"
        />
        <p>
          warning Cars45 is committed to keeping your information safe
        </p>
      </div>
      <div
        class="contact-line"
      />
      <h1>
        Contact Team F.A.S.T
      </h1>
      <p>
        <strong>
          To Join our Dealer Program
        </strong>
         
        <br />
        <span>
          Contact: 
          <br />
          4TH FLOOR, CIPM BUILDING, 
          <br />
          CIPM AVENUE, ALAUSA, IKEJA, LAGOS, NIGERIA 
          <br />
          +234 818 984 0160, +234 908 747 4947, +234 908 769 5935 
          <br />
          franchise@cars45.com
        </span>
      </p>
    </div>
  </div>
  <div>
    <div
      class="footer-one border-top"
    >
      <div
        class="footer-menu"
      >
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col-9 col-md-7"
            >
              <div
                class="row"
              >
                <div
                  class="col col-md-2 d-none d-md-block"
                >
                  <a
                    href="/about"
                  >
                    About Us
                  </a>
                </div>
                <div
                  class=" col col-md-2"
                >
                  <a
                    href="/"
                  >
                    Terms Of Use
                  </a>
                </div>
                <div
                  class="col col-md-2"
                >
                  <a
                    href="/"
                  >
                    Privacy Policy
                  </a>
                </div>
                <div
                  class="col col-md-1"
                >
                  <a
                    href="/"
                  >
                    FAQs
                  </a>
                </div>
                <div
                  class="col col-md-2"
                >
                  <a
                    href="/loan"
                  >
                    Financing
                  </a>
                </div>
                <div
                  class="col col-md-3 d-none d-md-block"
                >
                  <a
                    href="/franchise"
                  >
                    Become A Franchise
                  </a>
                </div>
              </div>
            </div>
            <div
              class="col-1 col-md-5"
            >
              <div
                class="row"
              >
                <div
                  class="col-md-6 text-center"
                >
                  <a
                    href="mailto:help@car45.com"
                  >
                    help@car45.com
                  </a>
                </div>
                <div
                  class="col-md-6 text-right d-none d-md-block"
                >
                  <div
                    class="socials"
                  >
                    <a
                      href=" http://facebook.com/cars45.NG"
                      target="_blank"
                    >
                      <img
                        alt="facebook"
                        class="facebook"
                        src="https://storage.googleapis.com/cars45-web-bucket/fb.svg"
                      />
                    </a>
                    <a
                      href="https://www.instagram.com/cars45ng/"
                      target="_blank"
                    >
                      <img
                        alt="instagram"
                        src="https://storage.googleapis.com/cars45-web-bucket/insta.svg"
                      />
                    </a>
                    <a
                      href="https://twitter.com/cars45ng"
                      target="_blank"
                    >
                      <img
                        alt="twitter"
                        src="https://storage.googleapis.com/cars45-web-bucket/tw.svg"
                      />
                    </a>
                    <a
                      href="/youtube.com"
                      target="_blank"
                    >
                       
                      <img
                        alt="youtube"
                        src="https://storage.googleapis.com/cars45-web-bucket/yt.svg"
                      />
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="footer-section-one"
      >
        <div
          class="container"
        >
          <div
            class="row info"
          >
            <div
              class="col-md-7"
            >
              <p>
                Cars45 brand provides consumers with the opportunity to buy cars online and offline. Car dealers and end-users can now have access to a rich variety of cars that have been properly verified and checked, accompanied by a standard inspection report. What we are bringing to the market isn't just availability but access to cars with the right information such that consumers can make the right decision. Cars45 currently has its operations spread across Lekki, Ikeja, Ikorodu, Oshodi and Amuwo areas of Lagos, Ibadan, Kano, Port-Harcourt, and Abuja.
              </p>
            </div>
            <div
              class="col-md-5 subscribe"
            >
              <p>
                SUBSCRIBE TO GET NOTIFIED
                <br />
                 ON GREAT OFFERS ON CARS45
              </p>
              <form
                class="form-inline"
              >
                <div
                  class="form-group mr-4 mb-2"
                >
                  <input
                    class="form-control"
                    id="email"
                    name="email"
                    placeholder="Your email address"
                    type="email"
                  />
                </div>
                <button
                  class="btn btn-primary mb-2 teal-button"
                  type="submit"
                >
                  SUBSCRIBE
                </button>
              </form>
            </div>
          </div>
          <div
            class="row ssl"
          >
            <div
              class="col-md-6"
            >
              <p>
                SSL Protection | Buy with confidence
              </p>
              <p>
                buy.cars45.com has been security-scanned and is 100% protected by the Secure Socket Layer certificate. All files and data sent over this site are fully encrypted from third-party access.
              </p>
            </div>
            <div
              class="col-md-6"
            >
              <div
                class="socials"
              >
                <a
                  href="/facebook.com"
                >
                  <img
                    alt="facebook"
                    class="facebook"
                    src="https://storage.googleapis.com/cars45-web-bucket/facebook.svg"
                  />
                </a>
                <a
                  href="/instagram.com"
                >
                  <img
                    alt="instagram"
                    src="https://storage.googleapis.com/cars45-web-bucket/instagram.svg"
                  />
                </a>
                <a
                  href="/twitter.com"
                >
                  <img
                    alt="twitter"
                    src="https://storage.googleapis.com/cars45-web-bucket/twitter.svg"
                  />
                </a>
                <a
                  href="/youtube.com"
                >
                   
                  <img
                    alt="youtube"
                    src="https://storage.googleapis.com/cars45-web-bucket/youtube.svg"
                  />
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="footer-section-two"
      >
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col-md-6"
            >
              <ul>
                <li>
                  <a
                    href="/"
                  >
                    Terms & Conditions
                  </a>
                </li>
                <li>
                  <a
                    href="/"
                  >
                    Privacy
                  </a>
                </li>
              </ul>
            </div>
            <div
              class="col-md-6 text-center text-md-right"
            >
              <p>
                2021Cars45.com. ©All rights reserved
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
